composer.json

```
@@ -10,14 +10,16 @@
         }
     ],
     "require": {
+        "php": ">=7.0",
         "friendsofphp/php-cs-fixer": "^2.17"
     },
     "require-dev": {
         "roave/security-advisories": "dev-master"
     },
     "autoload": {
         "psr-4": {
-            "EzSystems\\EzPlatformCodeStyle\\": "src/lib/"
+            "EzSystems\\EzPlatformCodeStyle\\": "src/legacy/",
```

```suggestion
            "EzSystems\\EzPlatformCodeStyle\\": "src/bc/",
```

seems to be a  better naming choice.

---

Fixed in 4eba03e5.